Documentation/network/servicemesh/tls-default-certificate.rst

```diff
@@ -0,0 +1,52 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+**********************************
+Defaults certificate for Ingresses
+**********************************
+
+Cilium can use a default certificate for ingresses without ``.spec.tls[].secretName`` set.
+
+Prerequisites
+#############
+
+* Cilium must be configured with Kubernetes Ingress Support.
+  Please refer to :ref:`Kubernetes Ingress Support <gs_ingress>` for more details.
+
+Installation
+############
+
+.. tabs::
+
+    .. group-tab:: Helm
+
+        Defaults certificate for Ingresses can be enabled with helm flags
+        ``ingressController.defaultSecretNamespace`` and
+        ``ingressController.defaultSecretName```
+        set as true. Please refer to :ref:`k8s_install_helm` for a fresh installation.
+
+        .. parsed-literal::
+
+            $ helm upgrade cilium |CHART_RELEASE| \\
+                --namespace kube-system \\
+                --reuse-values \\
+                --set ingressController.defaultSecretNamespace=kube-system \\
+                --set ingressController.defaultSecretName=default-cert \\
+
+            $ kubectl -n kube-system rollout restart deployment/cilium-operator
+            $ kubectl -n kube-system rollout restart ds/cilium
+
+    .. group-tab:: Cilium CLI
+
+        .. include:: ../../installation/cli-download.rst
+
+        Cilium Ingress Controller can be enabled with the below command
+
+        .. code-block:: shell-session
+
+            $ cilium install \\
+                --helm-set ingressController.defaultSecretNamespace=kube-system \\
+                --helm-set ingressController.defaultSecretName=default-cert
```

Comment:
Can you just please remove one of the double backslashes on this block? They are necessary in the `.. parsed-literal::` block, where the contents is parsed, but not in the `.. code-block` where we don't need to escape them to prevent them from being interpreted.

```suggestion
            $ cilium install \
                --helm-set ingressController.defaultSecretNamespace=kube-system \
                --helm-set ingressController.defaultSecretName=default-cert
```

Reply:
Done.